src/main/kotlin/uk/gov/justice/digital/hmpps/hmppschallengesupportinterventionplanapi/resource/InvestigationsController.kt

```diff
@@ -44,7 +45,53 @@ import java.util.UUID
 class InvestigationsController(
   private val investigationService: InvestigationService,
 ) {
+
   @ResponseStatus(HttpStatus.CREATED)
+  @PostMapping("/{recordUuid}/referral/investigation")
+  @PutMapping("/{recordUuid}/referral/investigation")
```

Comment:
both post and put mapping for this endpoint? does this put mapping override the upsert endpoint?